test/register_test.jl

```diff
@@ -7,36 +8,64 @@ using REALMSupport.RegisterCore: CenterIndexedArray
     C = outer[5:20, 1:16]
```

Comment:
Are `A`, `B`, and `C` mimics of a 2d object under translation? That is, `outer` is a ground-truth 2d image, and you're mimicking an ROI under displacement? If so, since REALM collects images on a skew, wouldn't you have to compute the mismatch under that skew? In other words, if the light sheet is tilted at angle `θ` relative to vertical, pixel spacing is `Δp`, and the spacing between adjacent scan planes is `Δs`, then `img[i, j]` (where `i` is "vertical" and `j` is "horizontal") is collected at position
```julia
x = Δs * j + sin(θ) * Δp * i
y = cos(θ) * Δp * i
```
(if I've done my math correctly, but it's worth checking). To me it seems that you want to sample from `A`, `B`, and `C` using this transformation first, before you compute `mm`.